containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/mariadb-client-install.sh

```diff
@@ -15,7 +15,7 @@ MARIADB_VERSION=${DDEV_DATABASE#*:}
 
 # Configure the correct repository for mariadb
 set -x
-log-stderr.sh --timeout "${START_SCRIPT_TIMEOUT:-30}" mariadb_repo_setup --mariadb-server-version="mariadb-${MARIADB_VERSION}" --skip-maxscale --skip-tools --skip-key-import || exit $?
+log-stderr.sh --timeout "${START_SCRIPT_TIMEOUT:-30}" mariadb_repo_setup --mariadb-server-version="mariadb-${MARIADB_VERSION}" --skip-maxscale --skip-tools --os-type=debian --os-version=bookworm || exit $?
```

Comment:
`--skip-key-import` should be here because we already have it in the base image, with skip it's faster.

Reply:
Yes, it worked during Dockerfile build, but when I tested it manually inside ddev-webserver it failed due to lack of key. I didn't study it very carefully, but the key seemed not to be installed. My test technique was to run `sudo /usr/local/bin/mariadb_repo_setup --mariadb-server-version="mariadb-11.8" --skip-maxscale --skip-tools --os-type=debian --os-version=bookworm  ` inside `ddev ssh` - it succeeded but then the `sudo apt install mariadb-client` failed with no key.

Reply:
Will test again, maybe my approach was wrong.